### DISCUSSION TITLE
Shizuku

### DISCUSSION THREAD

**Discussion by fgpizzetti** [d4rken-org/sdmaid-se#675] [General]
For the ordinary user, Shizuku is too complicated to be managed. In my humble opinion, it wpuld be better to find an alternative...

**Comment by d4rken** [comment]
You don't have to use Shizuku. Just like any other set up step, it's optional. Complete as many set up steps as you can or as you like.

The alternative to Shizuku is to root your device, if you find another alternative let me know.
